hub/powertoys/run.md

```diff
@@ -44,7 +44,7 @@ The following general options are available for PowerToys Run in the PowerToys s
 | Ignore shortcuts in Fullscreen mode | When in full-screen (F11), PowerToys Run won't be engaged with the shortcut |
 | Maximum number of results | Maximum number of results shown without scrolling |
 | Clear the previous query on launch | When launched, previous searches will not be highlighted |
-| Preferred monitor position | If multiple monitors are in use, PowerToys Run can be launched on the desired monitor:<br />- Primary monitor<br />- Monitor with mouse cursor<br />- Monitor with focused window |
+| Preferred monitor position | If multiple monitors are in use, PowerToys Run can be launched on the desired monitor:<br />- Primary display<br />- Display with mouse cursor<br />- Display with focused window |
```

Comment:
Hopefully this fixes translation

Reply:
If translation issue is about _verb versus noun_: unfortunately, "monitor", "display", and "screen" can all be both.

Reply:
But I hope that with display it is correct/better.